src/ctelnet.cpp

```diff
@@ -964,26 +964,39 @@ QString cTelnet::getNewEnvironValueSystemType()
     QString systemType;
 
     // "SYSTEMTYPE" Inspired by https://www.rfc-editor.org/rfc/rfc1340.txt
-#if (defined(Q_OS_MAC) || defined(Q_OS_MACOS))
```

Comment:
BTW `Q_OS_MAC` is, according to the Qt documentation:
> Deprecated synonym for `Q_OS_DARWIN`.  Do not use.

In turn, `Q_OS_DARWIN` is:
> Defined on Darwin-based operating systems such as macOS, iOS, watchOS, and tvOS.

so this is NOT really the one to use - instead it should (and I have changed the test to) use `Q_OS_MACOS`.